### PR TITLE
test: fix tests for op return

### PR DESCRIPTION
**Pull request by chedieck**
Description
---
The tests for opReturn were still checking for some old behavior (where the paymentId was generated automatically).

This PR fixes those tests, and also adds the workflow so github also run the tests for every PR.

Test plan
---
`yarn test` should run without failures and this PR should have a single github action associated showing that.

### CODE REVIEW

**Review by Klakurka — CHANGES_REQUESTED** [review]
I get this in master too but shouldn't this run?

image

File: .github/workflows/pull-request-integration-test.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Run tests
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  test:
+    name: Run unittests
```
Comment (lissavxo) [inline .github/workflows/pull-request-integration-test.yml:8]: Run unittests -> Run unit tests

**Review by lissavxo — CHANGES_REQUESTED** [review]

**Review by Klakurka — APPROVED** [review]

File: .github/workflows/pull-request-integration-test.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install base dependencies
+        run: |
+          yarn
+          cd react
+          yarn
+
+      - name: Run react unittests
```
Comment (Klakurka) [inline .github/workflows/pull-request-integration-test.yml:21]: Should this be `unit tests` too?

**Review by Klakurka — APPROVED** [review]

**Review by lissavxo — APPROVED** [review]
looks good

**Review by Klakurka — APPROVED** [review]

**Pull request merged**